lib/browser/api/net.js

```diff
@@ -53,18 +53,26 @@ class IncomingMessage extends Readable {
 
   get headers () {
     const filteredHeaders = {}
-    const rawHeaders = this._responseHead.headers
-    Object.keys(rawHeaders).forEach(header => {
-      if (header in filteredHeaders && discardableDuplicateHeaders.has(header)) {
+    const { rawHeaders } = this._responseHead
+    rawHeaders.forEach(header => {
+      if (filteredHeaders.hasOwnProperty(header.key) && discardableDuplicateHeaders.has(header.key)) {
```

Comment:
Ugh, I'm sorry about JavaScript, but actually this has to be written:
```suggestion
      if (Object.prototype.hasOwnProperty.call(filteredHeaders, header.key) && discardableDuplicateHeaders.has(header.key)) {
```

otherwise a header called `hasOwnProperty` will break this code.

Reply:
👍 Always learn something new about JavaScript when I thought I knew it enough.